modules/services/mpd-discord-rpc.nix

```diff
@@ -0,0 +1,50 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.mpd-discord-rpc;
+  tomlFormat = pkgs.formats.toml { };
+  configFile = tomlFormat.generate "config.toml" cfg.settings;
+
+in {
+  meta.maintainers = [ maintainers.kranzes ];
+
+  options.services.mpd-discord-rpc = {
+    enable = mkEnableOption "Whether to enable the mpd-discord-rpc service";
+
+    settings = mkOption {
+      type = tomlFormat.type;
+      default = {
+        id = 677226551607033903;
```

Comment:
seems suspicious to have such a hard-coded value here

Reply:
That's the default discord application ID it runs through.